docs/sql-migration-guide.md

```diff
@@ -56,6 +56,8 @@ license: |
   In Spark 3.1 and earlier, table refreshing leaves dependents uncached.
 
   - In Spark 3.2, the usage of `count(tblName.*)` is blocked to avoid producing ambiguous results. Because `count(*)` and `count(tblName.*)` will output differently if there is any null values. To restore the behavior before Spark 3.2, you can set `spark.sql.legacy.allowStarWithSingleTableIdentifierInCount` to `true`.
+ 
+  - In Spark 3.2, a `SHOW TBLPROPERTIES` clause's output both show `key` and `value` columns whether `propertyKey` is defined. In Spark 3.1 and earlier, a `SHOW TBLPROPERTIES` clause's output only show a `value` column when `propertyKey` is defined.
```

Comment:
`propertyKey` is also an internal variable name?

Reply:
"output both show `key` and `value` columns" => "output shows `key` and `value` columns"

Reply:
> `propertyKey` is also an internal variable name?

Yea ==

Reply:
> "output both show `key` and `value` columns" => "output shows `key` and `value` columns"

Done

Reply:
> `propertyKey` is also an internal variable name?

Updated to  `whether you specify property key`. How about current? also ping @cloud-fan